src/redux/walletconnect.js

```diff
@@ -278,6 +276,7 @@ export const walletConnectOnSessionRequest = (uri, callback) => async (
         // We need to add a timeout in case the bridge is down
         // to explain the user what's happening
         timeout = setTimeout(() => {
```

Comment:
on ios if the wc callback is received correctly we don't get here at all since there's a clearTimeout inside the wc callback, on android we always get here the clearTimeout doesn't work and also the `meta` object was always null, so I'm getting it from navigation

Reply:
this is the callback https://github.com/rainbow-me/rainbow/pull/3189/files#diff-bad40338fc8c2d94666bcfaef38a3ce284d36a497a5e1b7605e817e43bc0b95eR218